app/api/views/route.ts

```
@@ -586,9 +587,15 @@ export async function POST(request: NextRequest) {
 
         if (documentVersion.type === "sheet") {
           if (useAdvancedExcelViewer) {
-            documentVersion.file = documentVersion.file.includes("https://")
-              ? documentVersion.file
-              : `https://${process.env.NEXT_PRIVATE_ADVANCED_UPLOAD_DISTRIBUTION_HOST}/${documentVersion.file}`;
+            if (documentVersion.file.includes("https://")) {
+              documentVersion.file = documentVersion.file;
```

Biome (1.9.4)

[error] file is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)